Dockerfile.dapper

```diff
@@ -18,11 +18,16 @@ ENV ARCH $DAPPER_HOST_ARCH
 
 # mtools and dosfstools are requirements for luet-makeiso >= 0.4.0 to build hybrid ISO.
 RUN zypper -n rm container-suseconnect && \
-    zypper -n install go1.18 git curl docker gzip tar wget zstd squashfs xorriso awk jq mtools dosfstools
+    zypper -n install go1.18 git curl docker gzip tar wget zstd squashfs xorriso awk jq mtools dosfstools unzip
 RUN curl -sfL https://github.com/mikefarah/yq/releases/download/v4.21.1/yq_linux_${ARCH} -o /usr/bin/yq && chmod +x /usr/bin/yq
 RUN if [ "${ARCH}" == "amd64" ]; then \
         curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.49.0; \
     fi
+
+RUN zypper addrepo http://download.opensuse.org/distribution/leap/15.3/repo/oss/ oss && \
```

Comment:
The base is bci-base:15.4. Should we leap 15.4 here too?

Reply:
i will make the changes